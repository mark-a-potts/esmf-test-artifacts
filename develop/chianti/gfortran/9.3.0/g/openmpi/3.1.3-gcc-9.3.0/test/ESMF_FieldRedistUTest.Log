build time -- 2021-04-23 12:43:04.760193
20210423 125001.299 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 125001.299 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 125001.299 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 125001.299 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 125001.299 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 125001.299 INFO             PET0 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 125001.299 INFO             PET0 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 125001.299 INFO             PET0 ESMF_COMM                   : openmpi
20210423 125001.299 INFO             PET0 ESMF_MOAB                   : enabled
20210423 125001.299 INFO             PET0 ESMF_LAPACK                 : enabled
20210423 125001.299 INFO             PET0 ESMF_NETCDF                 : enabled
20210423 125001.299 INFO             PET0 ESMF_PNETCDF                : disabled
20210423 125001.299 INFO             PET0 ESMF_PIO                    : enabled
20210423 125001.299 INFO             PET0 ESMF_YAMLCPP                : enabled
20210423 125001.299 INFO             PET0  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20210423 125001.299 INFO             PET0  NUMBER_OF_PROCESSORS           4
20210423 125001.300 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.301 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.301 INFO             PET0  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20210423 125001.301 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.302 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.302 INFO             PET0  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20210423 125001.303 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.304 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.304 INFO             PET0  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20210423 125001.309 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.315 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.316 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20210423 125001.320 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.328 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.328 INFO             PET0  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20210423 125001.330 INFO             PET0 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.331 INFO             PET0 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.331 INFO             PET0  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20210423 125001.331 INFO             PET0  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20210423 125001.331 INFO             PET0 Finalizing ESMF
20210423 125001.299 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 125001.299 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 125001.299 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 125001.299 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 125001.299 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET1 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 125001.299 INFO             PET1 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 125001.299 INFO             PET1 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 125001.299 INFO             PET1 ESMF_COMM                   : openmpi
20210423 125001.299 INFO             PET1 ESMF_MOAB                   : enabled
20210423 125001.299 INFO             PET1 ESMF_LAPACK                 : enabled
20210423 125001.299 INFO             PET1 ESMF_NETCDF                 : enabled
20210423 125001.299 INFO             PET1 ESMF_PNETCDF                : disabled
20210423 125001.299 INFO             PET1 ESMF_PIO                    : enabled
20210423 125001.299 INFO             PET1 ESMF_YAMLCPP                : enabled
20210423 125001.299 INFO             PET1  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20210423 125001.299 INFO             PET1  NUMBER_OF_PROCESSORS           4
20210423 125001.300 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.301 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.301 INFO             PET1  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20210423 125001.301 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.302 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.302 INFO             PET1  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20210423 125001.303 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.304 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.304 INFO             PET1  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20210423 125001.310 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.315 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.316 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20210423 125001.321 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.328 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.328 INFO             PET1  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20210423 125001.330 INFO             PET1 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.331 INFO             PET1 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.331 INFO             PET1  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20210423 125001.331 INFO             PET1  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20210423 125001.331 INFO             PET1 Finalizing ESMF
20210423 125001.299 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 125001.299 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 125001.299 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 125001.299 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 125001.299 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET2 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 125001.299 INFO             PET2 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 125001.299 INFO             PET2 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 125001.299 INFO             PET2 ESMF_COMM                   : openmpi
20210423 125001.299 INFO             PET2 ESMF_MOAB                   : enabled
20210423 125001.299 INFO             PET2 ESMF_LAPACK                 : enabled
20210423 125001.299 INFO             PET2 ESMF_NETCDF                 : enabled
20210423 125001.299 INFO             PET2 ESMF_PNETCDF                : disabled
20210423 125001.299 INFO             PET2 ESMF_PIO                    : enabled
20210423 125001.299 INFO             PET2 ESMF_YAMLCPP                : enabled
20210423 125001.299 INFO             PET2  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20210423 125001.299 INFO             PET2  NUMBER_OF_PROCESSORS           4
20210423 125001.300 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.301 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.301 INFO             PET2  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20210423 125001.301 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.302 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.302 INFO             PET2  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20210423 125001.303 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.304 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.304 INFO             PET2  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20210423 125001.309 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.315 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.316 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20210423 125001.320 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.328 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.328 INFO             PET2  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20210423 125001.330 INFO             PET2 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.331 INFO             PET2 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.331 INFO             PET2  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20210423 125001.331 INFO             PET2  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20210423 125001.331 INFO             PET2 Finalizing ESMF
20210423 125001.299 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210423 125001.299 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210423 125001.299 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210423 125001.299 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210423 125001.299 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210423 125001.299 INFO             PET3 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_05-2-g3f3d15a0c7
20210423 125001.299 INFO             PET3 ESMF library build date/time: "Apr 23 2021" "12:42:18"
20210423 125001.299 INFO             PET3 ESMF library build location : /home/mpotts/gfortran_9.3.0_openmpi_g_develop
20210423 125001.299 INFO             PET3 ESMF_COMM                   : openmpi
20210423 125001.299 INFO             PET3 ESMF_MOAB                   : enabled
20210423 125001.299 INFO             PET3 ESMF_LAPACK                 : enabled
20210423 125001.299 INFO             PET3 ESMF_NETCDF                 : enabled
20210423 125001.299 INFO             PET3 ESMF_PNETCDF                : disabled
20210423 125001.299 INFO             PET3 ESMF_PIO                    : enabled
20210423 125001.299 INFO             PET3 ESMF_YAMLCPP                : enabled
20210423 125001.299 INFO             PET3  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
20210423 125001.299 INFO             PET3  NUMBER_OF_PROCESSORS           4
20210423 125001.300 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.301 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.301 INFO             PET3  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
20210423 125001.301 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.302 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.302 INFO             PET3  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
20210423 125001.303 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.304 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.304 INFO             PET3  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
20210423 125001.309 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.315 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.316 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
20210423 125001.320 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.328 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.328 INFO             PET3  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
20210423 125001.330 INFO             PET3 ESMF_FieldRedistStoreNF(): Just entered routine.
20210423 125001.331 INFO             PET3 ESMF_FieldRedistStoreNF(): Leaving routine.
20210423 125001.331 INFO             PET3  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
20210423 125001.331 INFO             PET3  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
20210423 125001.331 INFO             PET3 Finalizing ESMF
