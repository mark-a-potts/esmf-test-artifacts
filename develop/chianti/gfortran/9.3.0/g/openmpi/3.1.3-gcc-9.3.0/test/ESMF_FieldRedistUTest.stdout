build time -- 2021-04-23 12:43:04.760193
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_FieldRedistUTest.F90, line 57
  NUMBER_OF_PROCESSORS           4
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test, ESMF_FieldRedistUTest.F90, line 72
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist basic test with compliant fields, ESMF_FieldRedistUTest.F90, line 79
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist 3d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 86
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos, ESMF_FieldRedistUTest.F90, line 98
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist congruent 5d fields with ungridded bounds and halos  srcToDstTransposeMap, ESMF_FieldRedistUTest.F90, line 111
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PASS  FieldRedist on Fields built on Meshes, ESMF_FieldRedistUTest.F90, line 119
  Ending Test, file ESMF_FieldRedistUTest.F90, line 122
  PET           0  Test Elapsed Time    33.579999999999998       msec.
  PET           1  Test Elapsed Time    33.861000000000004       msec.
  PET           3  Test Elapsed Time    33.704000000000001       msec.
  PET           2  Test Elapsed Time    33.920999999999999       msec.
